src/base/static/components/input-form/index.js

```diff
@@ -47,10 +49,6 @@ class InputForm extends Component {
     };
   }
 
-  componentDidMount() {
-    this.props.map.on("dragend", this.handleDragEnd.bind(this));
```

Comment:
I think we can delete `this.handleDragEnd` now - but do we need to set `isMapPositioned` when the map is dragged?